ngraph_bridge/CMakeLists.txt

```diff
@@ -30,24 +30,24 @@ message(STATUS "Shared Link Flags: ${CMAKE_SHARED_LINKER_FLAGS}")
 #-----------------------------------------------------------------------------------------------
 set(SRC
    api.cc
+   assign_clusters.cc
    backend.cc
    backend_manager.cc
+   cluster_manager.cc
+   deassign_clusters.cc
+   encapsulate_clusters.cc
    executable.cc
    ie_tensor.cc
    kernels/ngraph_encapsulate_op.cc
-   ngraph_assign_clusters.cc
+   mark_for_clustering.cc
    ngraph_builder.cc
-   ngraph_cluster_manager.cc
    ngraph_conversions.cc
```

Comment:
Was this intentionally left out?

Reply:
Yes, I wasn't too stringent about it and left out the cases where it makes sense. The unchanged files are using the nGraph API and, for instance, it is an "nGraph builder" or those are "conversions related to nGraph".